docs/tutorials/gymnasium_basics/load_quadruped_model.md

```diff
@@ -0,0 +1,245 @@
+	3. Tweak the environment reward parameters.
+	4. Tweak the environment observation parameters.
+3. Train an agent to locomote your robot.
+
+
+The reader is expected to be familiar with the `Gymnasium` API & library, the basics of robotics, and the included `Gymnasium/MuJoCo` environments and the robot model they use, familiarity with the **MJCF** file model format and the `MuJoCo` simulator is not required but is recommended.
+
+Setup
+------
+We will need `gymnasium>=1.0.0`.
+
+```sh
+pip install "gymnasium>=1.0.0"
+```
+
+Step 0.1 - Downloading Our Robot Model
+-------------------------
+In this tutorial we will load the [Unitree Go1](
+https://github.com/google-deepmind/mujoco_menagerie/blob/main/unitree_go1/README.md) robot from the excellent robot model collection [MuJoCo Menagerie](https://github.com/google-deepmind/mujoco_menagerie).
+![Unitree Go1 robot in a flat terrain scene](https://github.com/google-deepmind/mujoco_menagerie/blob/main/unitree_go1/go1.png?raw=true)
+
+`Go1` is a quadruped robot, controlling it to locomote is a significant learning problem, much harder than the `Gymnasium/MuJoCo/Ant` environment.
+
+We can download the whole MuJoCo Menagerie collection (which includes `Go1`),
+```sh
+git clone https://github.com/google-deepmind/mujoco_menagerie.git
+```
+You can use any other quadruped robot with this tutorial, just adjust the environment parameter values for your robot.
+
+
+Step 1 - Load the model
+-------------------------
+To load the model, all we have to do is use the `xml_file` argument with the `Ant-v5` framework.
+
+```py
+import gymnasium
+import numpy as np
+env = gymnasium.make('Ant-v5', xml_file='./mujoco_menagerie/unitree_go1/scene.xml')
+```
+
+Although this is enough to load the model, we will need to tweak some environment parameters to get the desired behavior for our environment, for now we will also explicitly set the simulation, termination, reward and observation arguments, which we will tweak in the next step.
+
+```py
+env = gymnasium.make(
+    'Ant-v5',
+    xml_file='./mujoco_menagerie/unitree_go1/scene.xml',
+    forward_reward_weight=0,
+    ctrl_cost_weight=0,
+    contact_cost_weight=0,
+    healthy_reward=0,
+    main_body=1,
+    healthy_z_range=(0, np.inf),
+    include_cfrc_ext_in_observation=True,
+    exclude_current_positions_from_observation=False,
+    reset_noise_scale=0,
+    frame_skip=1,
+    max_episode_steps=1000,
+)
+```
+
+
+Step 2 - Tweaking the Environment Parameters
+-------------------------
+Tweaking the environment parameters is essential to get the desired behavior for learning.
+In the following subsections, the reader is encouraged to consult the [documentation of the arguments](https://gymnasium.farama.org/main/environments/mujoco/ant/#arguments) for more detailed information.
+
+
+
+Step 2.1 - Tweaking the Environment Simulation Parameters
+-------------------------
+The arguments of interest are `frame_skip` & `reset_noise_scale` & `max_episode_steps`.
```

Comment:
The arguments of interest are `frame_skip`, `reset_noise_scale` and `max_episode_steps`.